server/config/config.go

```diff
@@ -180,8 +180,12 @@ func readConfig(fullConfigPath string) (*generalConfig, error) {
 		return nil, fmt.Errorf("Error reading config file: %s", err)
 	}
 
+	// expand environment variables
+	expandedFileBytes := []byte(os.ExpandEnv(string(fileBytes)))
+	fmt.Println(string(expandedFileBytes))
```

Comment:
Can you remove this so we're not printing the config to stdout?

Reply:
Hah, forgot to delete it. Will do.